.ci/run.sh

```diff
@@ -60,6 +60,10 @@ case "${CI_JOB}" in
 		sudo -E PATH="$PATH" CRI_RUNTIME="containerd" bash -c "make monitor"
 		echo "INFO: Running tracing test"
 		sudo -E PATH="$PATH" bash -c "make tracing"
+		if [[ "${CI_JOB}" =~ CC_CRI_CONTAINERD ]]; then
+			echo "INFO: Running Confidential Container tests"
+			sudo -E PATH="$PATH" CRI_RUNTIME="containerd" bash -c "make cc-containerd"
+		fi
 		;;
 	"CC_CRI_CONTAINERD"|"CC_SKOPEO_CRI_CONTAINERD"|"CC_CRI_CONTAINERD_CLOUD_HYPERVISOR"|"CC_SKOPEO_CRI_CONTAINERD_CLOUD_HYPERVISOR")
```

Comment:
I think the merge is bad here too. `CC_CRI_CONTAINERD` is in two case branches

Reply:
The change here didn't work either. You know have:
```
if [[ "${CI_JOB}" =~ CC_CRI_CONTAINERD ]]; then
	echo "INFO: Running Confidential Container tests"
	sudo -E PATH="$PATH" CRI_RUNTIME="containerd" bash -c "make cc-containerd"
fi
```
in a case branch that doesn't include either `CC_CRI_CONTAINERD` options. When making these merges remember that `CCv0` builds on top of `main`, so we want to minimise the differences between the branches.